### PR TITLE
Make the koji builds endpoint more consistent with the copr endpoint

### PR DESCRIPTION
**Pull request by IceWreck**

### CODE REVIEW

**Comment by IceWreck** [comment]
recheck

**Comment by IceWreck** [comment]
Dont know why/where black is failing. Its working fine locally

**Comment by csomh** [comment]
> Dont know why/where black is failing. Its working fine locally.

Yeah, I see this on other PRs, too, I'll have a look.

File: packit_service/service/api/koji_builds.py [hunk 1]
```diff
@@ -65,22 +81,44 @@ def get(self):
 class KojiBuildItem(Resource):
     @koji_builds_ns.response(HTTPStatus.OK, "OK, koji build details follow")
     @koji_builds_ns.response(
-        HTTPStatus.NO_CONTENT.value, "Koji build identifier not in db/hash"
+        HTTPStatus.NOT_FOUND.value, "No info about build stored in DB"
     )
     def get(self, id):
         """A specific koji build details. From koji_build hash, filled by worker."""
         builds_list = KojiBuildModel.get_all_by_build_id(str(id))
         if bool(builds_list.first()):
```
Comment (csomh) [inline packit_service/service/api/koji_builds.py:89]: Could you take the "early return" approach here? Return early with `NOT_FOUND` if there is nothing in `builds_list`.

And maybe the `bool()` is not necessary.

(I know this is not strictly related to this PR, but if you touch this part anyways, let's fix it :slightly_smiling_face:)

**Review by csomh — COMMENTED** [review]
Looks nice, thank you!

File: packit_service/service/api/koji_builds.py (hunk 1 above)
Comment (IceWreck) [reply packit_service/service/api/koji_builds.py:89]: Sure

**Review by csomh — APPROVED** [review]
Nice! Let's merge this! :smiley:

**Pull request merged**